.spelling

```diff
@@ -325,6 +326,7 @@ OpenSSL
 openssl
 OS-level
 Ostrowski
+overridable
```

Comment:
Try to avoid adding jargon when possible. My suggestion to makes the text clearer and makes this change unneeded.

Reply:
removed and changes as noted above.